src/partials/products-section.html

```diff
@@ -4,28 +4,43 @@
     <h2 class="products__title">PRODUCTS</h2>
     <ul class="products__list">
       <li class="products__list-item">
-        <h3>ice cream</h3>
-        <div>...</div>
-        <p>
+        <h3 class="products__subtitle">ice cream</h3>
+        <div class="products__suspension-dots"></div>
+        <p class="products__description">
           Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
           eiusmod tempor incididunt ut labore et
         </p>
+        <button class="products__btn" type="button">
+          <svg width="40px" height="40px" fill="#D41443">
+            <use href="/src/images/sprite.svg#arrow-right"></use>
+          </svg>
+        </button>
       </li>
       <li class="products__list-item">
-        <h3>ice coffee</h3>
-        <div>...</div>
-        <p>
+        <h3 class="products__subtitle">ice coffee</h3>
```

Comment:
aici practic fiecare lista are un subtitlu; pct de suspensi si descriere si buton
blocul parinte este products__list-item sau mai bine products__list deci din el faci titlul products__list-item; products__list-subtitle; products__list-description; products__list-suspension-dots (aici cred ca poti lasa dots sa mai micsorezi) si butonul care va fi independent caci il mai folosesti si in alta parte deci este oarecum global

Reply:
am modificat si aici, initial asa ma gandeam si eu sau sa fac un nou bloc de la li, multumesc de idei eu invat mult din asta